sw/device/lib/runtime/ibex.h

```diff
@@ -13,6 +13,9 @@
 #include "sw/device/lib/base/math.h"
 #include "sw/device/lib/base/stdasm.h"
 
+// IBEX_SPIN_FOR needs a dependency on check.h, but the build fails if a
+// dependency on sw_lib_testing_check is added.
```

Comment:
The check.h header still gets pulled in, which is all that really matters. This is a meson issue (meson seems to care about the order in which targets are defined so you can't use a target as a dep before it is defined, in terms of the order things are defined in meson.build files, and unfortunately there is a bit a of circular dependency here where we can't just move around `subdir` statements to solve). This won't be an issue in bazel, so I don't think its worth the time to even resolve this cosmetic issue. But good catch :)